src/services/item/repository.ts

```diff
@@ -234,15 +235,29 @@ export const ItemRepository = AppDataSource.getRepository(Item).extend({
       .getRawOne();
   },
 
-  async getOwn(memberId: string): Promise<Item[]> {
-    return this.createQueryBuilder('item')
+  async getOwn(
+    memberId: string,
+    page: number,
+    name: string,
+    all: boolean,
+    limit: number,
```

Comment:
I prefer the following
```suggestion 
type PaginationArgs = { 
      page: number,
      all: boolean,
      limit: number,
    }

  async getOwn(
    memberId: string,
    searchArgs: {
      name: string,
    },
    args: PaginationArgs
```